pyrogram/client/client.py

```diff
@@ -284,11 +295,12 @@ def start(self):
                 if self.bot_token is None:
                     self.authorize_user()
```

Comment:
...because you are relying on `self.is_bot = False` here. Better be explicit and set it to False here instead.